front/app/containers/Admin/pagesAndMenu/components/SectionFormWrapper.tsx

```diff
@@ -24,9 +24,11 @@ const SectionFormWrapper = ({
 }: Props) => {
   return (
     <>
-      <Box mb="16px">
-        <Breadcrumbs breadcrumbs={breadcrumbs} />
-      </Box>
+      {breadcrumbs && (
```

Comment:
Not sure what you mean. We add the check here around the wrapper (styles) + Breadcrumbs component so styles won't render if there are no breadcrumbs.

Reply:
Ah indeed I didn't think about the wrapper. Then only one point is left. The condition is different here and in the component (`!breadcrumbs || breadcrumbs.length === 0`). So, if `breadcrumbs` is an empty array, the wrapper will be rendered, but not the component.

Though not a big deal of course.

Reply:
Okay, removed the !breadcrumbs in the component, it's redundant with the required prop. We do need the check outside (for the Box). The length check inside the component doesn't matter atm, because the Box wrapper doesn't have styles that would be displayed, but it doesn't hurt to have it.